packages/server/src/utilities/rowProcessor/index.ts

```diff
@@ -182,8 +220,10 @@ export async function inputProcessing(
     clonedRow._rev = row._rev
   }
 
-  // handle auto columns - this returns an object like {table, row}
```

Comment:
Personally, I am not a fan of not having pure functions. It makes the code harder to read, as you never know if the input was modified or not

Reply:
The function wasn't pure to begin with, part of the reason I removed the return value. It was always modifying the original. It's clearer that that's happening now that it has no return value, in my opinion. 

I wouldn't mind converting it into a pure function if you'd prefer, I tend to prefer side-effect free functions as well. It does introduce a strictly unnecessary copy, though.